### PR TITLE
Adding dev names

### PR DESCRIPTION
**Pull request by erperezm**
Fixing dev names

### CODE REVIEW

**Comment by erperezm** [comment]
@paulperez-dev  From now you need to review all my code before merge the pull request.

File: patterns/state-machine/common_features_state.gd [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+#region common methods
+#execute before node start
+func start():
+	pass
+#execute after node end
+func end():
+	pass
+	
+#endregion
```
Comment (paulperez-dev) [inline patterns/state-machine/common_features_state.gd:18]: It is OK for these to be empty at this point?

File: scenes/player_test/player_test.gd [hunk 2]
```diff
@@ -0,0 +1,25 @@
+extends CharacterBody2D
```
Comment (paulperez-dev) [inline scenes/player_test/player_test.gd:1]: Test file. If not longer needed consider removing it.

File: scenes/player_test/states/left.gd [hunk 3]
```diff
@@ -0,0 +1,8 @@
+extends CommonFeaturesState
+
+
+func on_input(event):
+	# seria mejor usar el parametro _event para obtener la información del evento
```
Comment (paulperez-dev) [inline scenes/player_test/states/left.gd:5]: Why this was not changed?

File: scenes/player_test/states/left.gd [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+	if Input.is_action_just_pressed("ui_left"): 
+		state_machine.change_to("Right")
+		print("right")
```
Comment (paulperez-dev) [inline scenes/player_test/states/left.gd:8]: I think we won't use these actions in the final version

File: scenes/player_test/states/right.gd [hunk 5]
```diff
@@ -0,0 +1,8 @@
+extends CommonFeaturesState
+
+func on_input(event):
+	# seria mejor usar el parametro _event para obtener la información del evento
+	if Input.is_action_just_pressed("ui_right"): 
+		state_machine.change_to("Left")
+		
+		print("left")
```
Comment (paulperez-dev) [inline scenes/player_test/states/right.gd:8]: Same as above

File: scripts/test.gd
Comment (paulperez-dev) [inline scripts/test.gd:1]: Test empty file.

**Review by paulperez-dev — CHANGES_REQUESTED** [review]
I left some comments, please review.

File: scenes/player_test/player_test.gd (hunk 2 above)
Comment (erperezm) [reply scenes/player_test/player_test.gd:1]: Is used to demostrate how to use StateMachine

File: scenes/player_test/states/left.gd (hunk 3 above)
Comment (erperezm) [reply scenes/player_test/states/left.gd:5]: I need understand why first.

File: scenes/player_test/states/left.gd (hunk 4 above)
Comment (erperezm) [reply scenes/player_test/states/left.gd:8]: yep there is examples, i think we need other branch to upload these kilnd of example code.

**Comment by erperezm** [comment]
I decided to create another branch to up examples and leave the main clean.

**Pull request closed without merging**